src/sentry/static/sentry/app/stores/tooltipStore.tsx

```diff
@@ -0,0 +1,41 @@
+import Reflux from 'reflux';
+
+import Tooltip from 'app/components/tooltip';
```

Comment:
Is this circular import going to be an issue?

Reply:
Hmm good call, just need it for it's type, how do we normally deal with that?

Reply:
Hmm it might be ok if it's just the type?

Reply:
Switched it to just generically use components instead and have it just call setState to avoid the issue for now.